docs/src/starknet/script.md

```diff
@@ -13,6 +13,18 @@ have a `main` function in the module you want to run. `cast_std` docs can be fou
 Please note that **`sncast script` is in development**. While it is already possible to declare, deploy, invoke and call
 contracts from within Cairo, its interface, internals and feature set can change rapidly each version.
 
+> ⚠️⚠️ By default, the nonce for each transaction is being taken from the pending block ⚠️⚠️
+>
+> Some RPC nodes can be configured with higher poll itervals, which means they may return "older" nonces
+> in pending blocks, or even not be able to obtain pending blocks at all. When running scripts you see
+> error, you may need to manually set nonce and max_fee for transactions.
```

Comment:
```suggestion
> in pending blocks, or even not be able to obtain pending blocks at all. When running a script you see an
> error, you may need to manually set both nonce and max_fee for transactions.
```

Let's also state what's the error that might indicate this problem (wrong nonce afaik, right?)

Reply:
let's state the error that might suggest this is the case

Reply:
So I need a node without the pending block, to check what error will be returned

Reply:
That would be ideal, but I know it may be hard to do + it does not mean that this will be the error users see. That's why I'd suggest to generalise - state that if you see errors like XYZ, this might be the case that you are facing this problem. sounds good?